hypervisor/src/mshv/mod.rs

```diff
@@ -789,6 +805,8 @@ impl cpu::Vcpu for MshvVcpu {
                     let ghcb_msr = svm_ghcb_msr {
                         as_uint64: info.ghcb_msr,
                     };
+                    let ghcb = self.ghcb.as_ref().unwrap().0;
```

Comment:
Please leave a comment here saying why `unwrap` is okay.

Reply:
Added. Thanks :)